### ISSUE TITLE
GHS + PFO: pasting table from Google Sheets displays HTML comment

### ISSUE THREAD

**Issue by mmotyczynska** [ckeditor/ckeditor5#14131]
## 📝 Provide detailed reproduction steps (if any)

1. Add `PasteFromOffice` to `ghs-all-features` manual test.
2. Change config for GHS:
```
htmlSupport: {
	allow: [
		{
			name: /^(table|thead|tbody|td|th|tr|col|colgroup)$/,
			styles: true,
		},
	],
},
```
3. Paste a table from Google Sheets. 

### ✔️ Expected result

The table is displayed in the editor.

### ❌ Actual result

Besides the table, there is also an HTML comment displayed above.
![image](https://github.com/ckeditor/ckeditor5/assets/95343030/4b1b5f15-fb48-446e-939c-b26eb7756adf)

### ❓ Possible solution

_If you have ideas, you can list them here. Otherwise, you can delete this section._

## 📃 Other details

* Browser: …
* OS: …
* First affected CKEditor version: …
* Installed CKEditor plugins: …

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by filipsobol** [comment]
@ethan-rally This issue will be fixed in `v39.0.0`. I confused this task with another and accidentally changed the iteration from 64 (`v39`) to 63 (`v38`). I have changed the iteration back to 64 to properly reflect which version contains the fix.

**Comment by ethan-rally** [comment]
Thank you so much for the update. When it ( v39 ) is going to be released ? This issue was a blocker for us to upgrade v38, since v37 didn't have a issue like this. Do you have any ETA for v39 ?
